packages/et-mfr/src/Adaptor.js

```diff
@@ -0,0 +1,152 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils.js';
+
+/**
+ * State object
+ * @typedef {Object} ET-MFRHttpState
```

Comment:
I'm surprised if this is a legal state name - can we just call it `HttpState`? It doesn't need to be unique